.github/workflows/cypress-demo.yml

```diff
@@ -62,9 +62,9 @@ jobs:
           headless: true
           record: true
           parallel: true
-          config-file: cypress/config/ci.json
+          config-file: cypress/config/ci.config.ts
```

Comment:
Cypress config is now a JS or TS file. See https://docs.cypress.io/guides/references/configuration#Configuration-File